src/main/java/org/nguh/nguhcraft/NguhPaintings.kt

```diff
@@ -17,9 +17,10 @@ object NguhPaintings {
     val MAP = of("map")
     val RABO = of("rabo")
     val RAIL_DIAGRAM = of("rail_diagram")
+    val CHILLVANA_METRO = of("chillvana_metro")
```

Comment:
On that note, these are also supposed to be sorted

Reply:
alright